packages/webapp/pages/posts/[id]/edit.tsx

```diff
@@ -51,9 +51,10 @@ function EditPost(): ReactElement {
   } = useDiscardPost({ post });
   const {
     mutateAsync: onUpdatePost,
-    isLoading: isPosting,
+    isPending: isPosting,
```

Comment:
So isLoading is just isPending now? https://tanstack.com/query/v5/docs/framework/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading